src/types/big-integer.ts

```diff
@@ -26,7 +26,7 @@ export const fetchProcessing: TypeUtils.FetchProcessing<Types['Read']> = (
 	} else {
 		throw new Error('Fetched bigint is not valid: ' + typeof data);
 	}
-	return value;
+	return value.toString();
```

Comment:
I'd rather only do the `toString()` when it's necessary and avoid unnecessary conversions to BigInt

Reply:
Updated